simpletransformers/classification/classification_model.py

```diff
@@ -501,7 +501,7 @@ def train(
                             best_eval_metric = results[args.early_stopping_metric]
                             self._save_model(args.best_model_dir, optimizer, scheduler, model=model, results=results)
                         if best_eval_metric and args.early_stopping_metric_minimize:
-                            if results[args.early_stopping_metric] - best_eval_metric < args.early_stopping_delta:
+                            if best_eval_metric - results[args.early_stopping_metric] > args.early_stopping_delta:
```

Comment:
Aren't both these conditions the same? :thinking:

Reply:
The change is the same as: `if results[args.early_stopping_metric] - best_eval_metric < - args.early_stopping_delta:`

Reply:
Got it, my bad.